### ISSUE TITLE
Message Analyzer is dead

### ISSUE THREAD

**Issue by nischl** [MicrosoftDocs/windowsserverdocs#4696]
[Enter feedback here]

Since Message Analyzer is dead, how are we supposed to analyse these things?
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 69242460-9916-0cca-0858-a36f7f427991
* Version Independent ID: c154f1d7-ec73-2d8d-5fd8-7adf18b668a5
* Content: [Troubleshooting a Failover Cluster using Windows Error Reporting](https://docs.microsoft.com/en-us/windows-server/failover-clustering/troubleshooting-using-wer-reports)
* Content Source: [WindowsServerDocs/failover-clustering/troubleshooting-using-WER-reports.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/failover-clustering/troubleshooting-using-WER-reports.md)
* Product: **windows-server**
* Technology: **storage-failover-clustering**
* GitHub Login: @dcuomo
* Microsoft Alias: **vpetter**

**Comment by illfated** [comment]
I am not sure if you will get any answers here to your question, at least not anytime soon.

You may want to look elsewhere for answers, unless you have suggestions for improving one or more specific pages in the MS Docs repository.

Disclaimer: I am not affiliated with Microsoft or the Microsoft Docs teams here on GitHub.

The intention of this repository is to improve on the existing documentation by providing the users a way to report factual errors, improvement suggestions and request updates for new versions.
The MS Docs team members might not have time to answer your question, both due to the small size of the team and the big workload.

Try the following resources to get in touch with users who may be able to help you:
- TechNet forums: https://social.technet.microsoft.com/Forums/en-US/home
- MSDN: https://social.msdn.microsoft.com/Forums/en-US/home
- Microsoft Community: https://answers.microsoft.com/

**Comment by nischl** [comment]
I don't want to shoot the messenger, but....  you are basically saying Microsft does not care about the customers and the customer base should try to find everything out themselves? I expected something like this, but I still have some hope. 
Btw: I expect a product that is being sold having a good documentation. I am very disappointed by what Microsoft is delivering here not only in this specific topic but on docs.microsoft.com.

**Comment by illfated** [comment]
Sorry if my message gives the impression that Microsoft does not care.

I am merely trying to ask if this is meant to be a report that the document page needs to be changed, and if yes, then how do you recommend that it should be changed.

My other advices in my previous comment was meant to provide you with alternatives, in case you want to troubleshoot an issue on your side, especially if the issue is not proven to be happening with everybody else all over the world (or at least all those who use the exact same software configuration as you).

**Comment by illfated** [comment]
Also, if you are trying to troubleshoot a problem on your own computer, you may want to contact your nearest Microsoft Support department, to make sure that any undocumented issues are coexisting with the issue you are reporting here.

**Comment by nischl** [comment]
Yes, of course my report is here because the page needs to be changed! really, why else would I invest my time to put it up? 

The tool "message analyzer" that is referred to on the page is simply not available anymore. Microsoft just deleted it. While it is ok for them to do that - it's their software - we need a different approach now. So you either update the page with info on how to do it with tools that are actually available or you just delete the page and let that information die with it

**Comment by illfated** [comment]
Sorry for appearing to be slow and hard of understanding, there are quite a lot of users who open tickets here to ask for support to resolve issues local to their own computers.

Thank you for your feedback. Let us hope that the MS Docs team will soon let us know of any plans to change the page.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
